.github/workflows/build-admin-ui.yaml

```diff
@@ -74,3 +74,28 @@ jobs:
         with:
           name: admin-ui-ent
           path: ./ui/admin/dist/
+
+  build-admin-ui-hcp:
+    name: Build Admin UI HCP
+    needs: [dependencies]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        id: node-modules-cache
+        with:
+          path: '**/node_modules'
+          key: node-modules-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            node-modules-
+      - name: Build
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+        if: steps.node-modules-cache.outputs.cache-hit != 'true'
+      - run: yarn run build:ui:admin:enterprise
+      - name: Upload artifact Admin UI Enterprise
+        uses: actions/upload-artifact@v3
+        with:
+          name: admin-ui-ent
```

Comment:
Should this be renamed to hcp? We already have [admin-ui-ent](https://github.com/hashicorp/boundary-ui/pull/1221/files#diff-9107bf357ecdbe3083ccc32162dfa2d2826ac1fac3468e413fdb65a5cf3829f8R75) variant.

Reply:
Ahh thank you for the catch, that is definitely needed. I just pushed the update 🙏